### PR TITLE
Elo choquet branch: 

### PR DESCRIPTION
**Pull request by echoquet**
please include my Rover file!

### CODE REVIEW

File: Earth.txt [hunk 1]
```diff
@@ -6,4 +6,7 @@ plates that migrate across the surface over many millions of years. Earth's inte
 solid iron inner core, a liquid outer core that generates Earth's magnetic field, and a convecting mantle
 that drives plate tectonics.
 
-Taken from https://en.wikipedia.org/wiki/Earth
\ No newline at end of file
+Taken from https://en.wikipedia.org/wiki/Earth
+
+
+Just testing what happens if I also change the Earth file and try the pull requst
```
Comment (ivalaginja) [inline Earth.txt:12]: Are you sure you actually want to include this in this PR?

**Review by alexisyslau — COMMENTED** [review]
Sure ;) Go persy!

File: Earth.txt (hunk 1 above)
Comment (echoquet) [reply Earth.txt:12]: You can discard that

File: Earth.txt [hunk 2]
```diff
@@ -6,4 +6,7 @@ plates that migrate across the surface over many millions of years. Earth's inte
 solid iron inner core, a liquid outer core that generates Earth's magnetic field, and a convecting mantle
 that drives plate tectonics.
 
-Taken from https://en.wikipedia.org/wiki/Earth
\ No newline at end of file
+Taken from https://en.wikipedia.org/wiki/Earth
```
Comment (ivalaginja) [inline Earth.txt:9]: oh here too

File: Elo-Choquet_Rovers.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+Go persy! land on Mars!
```
Comment (ivalaginja) [inline Elo-Choquet_Rovers.txt:1]: this is nice!

**Review by ivalaginja — COMMENTED** [review]
I just have some comments on this

**Review by alexisyslau — APPROVED** [review]
Go! A brave rover!

**Pull request merged**